src/panoptes/pocs/focuser/serial.py

```diff
@@ -15,13 +16,55 @@ class AbstractSerialFocuser(AbstractFocuser):
     # known focuser devices & acts as a check against adaptors assigned to incorrect ports.
     _assigned_nodes = []
 
-    def __init__(self, *args, **kwargs):
+    def __init__(self, dev_node_pattern="", initial_position=None,
+                 serial_number_pattern="", *args, **kwargs):
```

Comment:
Default to `None` for unknown things.

```suggestion
    def __init__(self, dev_node_pattern=None, initial_position=None,
                 serial_number_pattern=None, *args, **kwargs):
```

Reply:
For the `serial_number_pattern`, I would create your own `re` object if given a string.